src/pages/landing/deriv-go/_banner.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import { Container, Flex } from 'components/containers'
+import device from 'themes/device.js'
+// svgs
+import TopBannerLogo from 'images/svg/deriv-go/deriv-go-logo.svg'
+import GooglePlayLogo from 'images/svg/deriv-go/google-play.svg'
+import { localize } from 'components/localization'
+import { Header, QueryImage } from 'components/elements'
+
+const query = graphql`
+    query {
+        deriv_go_banner: file(relativePath: { eq: "deriv-go/banner-img.png" }) {
+            ...fadeIn
+        }
+        deriv_go_banner_m: file(relativePath: { eq: "deriv-go/banner-img-m.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const MainWrapper = styled(Flex)`
+    background-color: var(--color-white);
+    flex-direction: column;
+    overflow: hidden;
+`
+const HeaderWrapper = styled(Flex)`
+    position: relative;
+    overflow: hidden;
+    height: 562px;
+    background: rgb(80, 133, 182);
+    background: linear-gradient(180deg, rgba(80, 133, 182, 1) 0%, rgba(30, 60, 87, 1) 100%);
+    clip-path: polygon(0 0, 100% 0%, 100% 85%, 0 100%);
+    @media ${device.desktopL} {
+        height: 720px;
+    }
+    @media ${device.tabletL} {
+        clip-path: polygon(0 0, 100% 0%, 100% 85%, 0 100%);
+        height: 632px;
+    }
+`
+const TopContent = styled(Flex)`
+    max-width: 996px;
+    padding: 0;
+    margin: 0 auto;
+    justify-content: flex-start;
+
+    @media ${device.laptopM} {
+        padding: 0 40px;
+    }
+    @media ${device.tabletL} {
+        max-width: 58.8rem;
+        flex-direction: column;
+        justify-content: center;
+        height: 367px;
+        padding: 0 16px;
+    }
+`
+const TopContentLeft = styled(Flex)`
+    flex-direction: column;
+    width: 486px;
+    margin-right: 126px;
+    justify-content: flex-start;
+    align-items: flex-start;
+    @media ${device.laptopM} {
+        margin-right: 80px;
+    }
+    @media ${device.tabletL} {
+        justify-content: center;
+        margin-right: 0;
+        width: 100%;
+    }
+`
+const TopContentLogo = styled.img`
+    width: 213px;
+    height: 28px;
+    margin-top: 32px;
+    margin-bottom: 71px;
```

Comment:
```suggestion
    margin: 32px 0 71px;
```

Reply:
It is intended,because the text should be align on left in desktop view.